contracts/Tokenomics.sol

```diff
@@ -1296,6 +1296,46 @@ contract Tokenomics is TokenomicsConstants {
         return true;
     }
 
+    /// @dev Updates inflation per second if .
+    function updateInflationPerSecond() external {
+        // Check for the contract ownership
+        if (msg.sender != owner) {
+            revert OwnerOnly(msg.sender, owner);
+        }
+
+        // Get current epoch length
+        uint256 curEpochLen = epochLen;
+        // Find if the year changes within the epoch
+        uint256 numYears = (block.timestamp + curEpochLen - timeLaunch) / ONE_YEAR;
+        // Revert if the year changes within the next epoch as it requires more complicated set of calculations
+        if (numYears > currentYear) {
+            revert Overflow(numYears, currentYear);
+        }
```

Comment:
hm - can you be more explicit what happens then?

Reply:
When the year changes, the complex re-calculation of inflation before the exact second year end and the rest of the epoch happens, so it's better to avoid to perform another change during that time.

Reply:
OK

Reply:
Please add as much detail in the contract comment for future